README.md

```diff
@@ -1,128 +1,213 @@
-# Realtime Client
+<br />
+<p align="center">
+  <a href="https://supabase.io">
+        <picture>
+      <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/supabase/supabase/master/packages/common/assets/images/supabase-logo-wordmark--dark.svg">
+      <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/supabase/supabase/master/packages/common/assets/images/supabase-logo-wordmark--light.svg">
+      <img alt="Supabase Logo" width="300" src="https://raw.githubusercontent.com/supabase/supabase/master/packages/common/assets/images/logo-preview.jpg">
+    </picture>
+  </a>
 
-Listens to changes in a PostgreSQL Database and via websockets.
+  <h1 align="center">Supabase Realtime Client</h1>
 
-This is for usage with Supabase [Realtime](https://github.com/supabase/realtime) server.
+  <h3 align="center">Send ephemeral messages with <b>Broadcast</b>, track and synchronize state with <b>Presence</b>, and listen to database changes with <b>Postgres Change Data Capture (CDC)</b>.</h2>
```

Comment:
Did you want this as an h3 or an h2 (different opening and closing tags)?

Reply:
@dannykng ah, thanks! i meant h3

Reply:
done